homeassistant/components/avreceiver/strings.json

```diff
@@ -0,0 +1,30 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "title": "Connect to your audio/video receiver",
+        "description": "Please enter the host name or IP address of the AV receiver",
+        "data": {
+          "host": "[%key:common::config_flow::data::host%]"
+        }
+      }
+    },
+    "abort": {
+      "single_instance_allowed": "[%key:common::config_flow::abort::single_instance_allowed%]",
+      "no_devices_found": "[%key:common::config_flow::abort::no_devices_found%]"
+    }
+  },
+  "options": {
+    "step": {
+      "init": {
+        "title": "Audio Video Receiver",
+        "description": "Add extra zones",
+        "data": {
+          "zone2": "Add Zone 2 (if available)",
+          "zone3": "Add Zone 3 (if available)",
+          "zone4": "Add Zone 4 (if available)"
```

Comment:
Can this be dynamic along with the options flow?

Reply:
I would not know how